js/controllers/MenuHandler.js

```diff
@@ -79,18 +82,41 @@
 
 
 	/**
-	 * Attach a class to any dropdown menus that will stray off-screen to align
-	 * them to the right edge of their parent
+	 * Check if submenus are straying off-screen and adjust as needed
 	 */
 	$.pkp.controllers.MenuHandler.prototype.setDropdownAlignment = function() {
-		var width = Math.max(
-				document.documentElement.clientWidth, window.innerWidth || 0);
+		var $this = $(this),
+			width = Math.max(
+				document.documentElement.clientWidth, window.innerWidth || 0),
```

Comment:
OTOH I think jslint will want another tab on these lines. If you haven't, pop in a PR to OMP with a submodule commit referring to this branch so that jslint gets run on it.